docs/sles-installation-guide.md

```diff
@@ -0,0 +1,72 @@
+# Installing Intel® Clear Containers 3.0 on SLES* version 12 SP1
+
+This document _only_ covers installing on a
+[SLES](https://www.suse.com/products/server/) 12 SP1 system. Although,
+this guide could be used with more recent Service Packs, it has not been
+tested.
+
+Note:
+
+If you are installing on a system that already has Clear Containers 2.x
+installed, first read [the upgrading document](upgrading.md).
+
+## Required Setup
+
+The installation requires the current user to run the `sudo` command
```

Comment:
Lines 15-16 suggested rewrite: "You are required to run the `sudo `command without specifying a password to set up the Clear Containers 3.0 installation. Verify this requirement with the following commands:"